src/objects.js

```diff
@@ -1,52 +1,84 @@
 const createPerson = (name, age) => {
-  // your code here
+    const user = {
+        name: (name),
+        age: (age)
+    }
+    return user
 };
 
 const getName = (object) => {
-  // your code here
+    return object.name
+        // your code here
 };
 
 const getProperty = (property, object) => {
-  // your code here
+    return object[property]
+        // your code here
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+    const a = !!object[property]
+    return a
```

Comment:
Again, no need for assignment here. Just return the `!!object[property]` expression :)